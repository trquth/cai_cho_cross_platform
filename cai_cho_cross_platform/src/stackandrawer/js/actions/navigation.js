import { BACK, TOGGLE_DRAWER } from './types';

export function back() {
  return { type: BACK };
}

export function toggleDrawer() {
  return { type: TOGGLE_DRAWER };
}
