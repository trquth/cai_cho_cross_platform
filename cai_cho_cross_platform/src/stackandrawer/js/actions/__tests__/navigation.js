import mockStore from 'redux-mock-store';
import * as actions from '../navigation';

const store = mockStore();

beforeEach(() => {
  store.clearActions();
});

it('BACK', () => {
  store.dispatch(actions.back());
  expect(store.getActions()).toMatchSnapshot();
});

it('TOGGLE_DRAWER', () => {
  store.dispatch(actions.toggleDrawer());
  expect(store.getActions()).toMatchSnapshot();
});
