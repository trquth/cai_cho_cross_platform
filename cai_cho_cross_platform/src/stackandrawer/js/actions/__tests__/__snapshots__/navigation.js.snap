// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BACK 1`] = `
Array [
  Object {
    "type": "BACK",
  },
]
`;

exports[`TOGGLE_DRAWER 1`] = `
Array [
  Object {
    "type": "TOGGLE_DRAWER",
  },
]
`;
